src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java

```diff
@@ -210,6 +213,22 @@ public void updateStatusById(UUID formId, FormStatus status) {
         formRepository.save(one);
     }
 
+    @Override
+    public Map<String, Object> getFileByFormId(UUID formId) {
+        MDCLog.putClassNameInMDC(SIMPLE_CLASS_NAME);
+        log.info("Try to download file by formId = {}", formId);
+        Form form = formRepository.findById(formId)
+                .orElseThrow(() -> new NotFoundException("Form with uuid = " + formId +
+                " Not Found in DB", "form.uuid.invalid"));
+        Map<String,Object> fileObjectMap = new HashMap<>();
```

Comment:
Prepare separate some container DTO instead of just map

Reply:
Fixed